src/utils/hooks/useFeatures/constants.ts

```diff
@@ -39,6 +40,7 @@ export const UI_FEATURES = [
 
 export const featuresConfigMapInitialState: IoK8sApiCoreV1ConfigMap = {
   data: {
+    [ADVANCED_CDROM_FEATURES]: 'false',
```

Comment:
the flag is not connected with that group of features flag (stored in `kubevirt-ui-features` config map)